### ISSUE TITLE
Bad Contrast for Download Zip Button on Grade Page

### ISSUE THREAD

**Issue by eddiekrystowski** [Submitty/Submitty#6503]
**Describe the bug**
The text for the download zip button is black in dark mode.

**Expected behavior**
The text should be white.

**To Reproduce**
1. Open a gradeable for grading
2. Scroll to bottom of page
3. See error

**Configuration**
 - MacOS, Chrome

**Screenshots**
![image](https://user-images.githubusercontent.com/28243927/119556674-49ab9a80-bd6d-11eb-90fa-d67114d5b8dd.png)
![image](https://user-images.githubusercontent.com/28243927/119556736-5a5c1080-bd6d-11eb-81c7-b1288c212ecf.png)
